canonical/ubuntu.com
# Ubuntu 21.10 (Impish Indri) release

## Done

- All the updates for the Ubuntu 21.10 release

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/download
    - http://0.0.0.0:8001/desktop/developers
    - http://0.0.0.0:8001/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
